packages/boxel-ui/addon/components/header/index.gts

```diff
@@ -81,6 +85,10 @@ const Header: TemplateOnlyComponent<Signature> = <template>
       display: flex;
       align-items: center;
     }
+    .header__icon {
```

Comment:
This is quite minor but with scoped CSS we don’t really need BEM for the most part, this could just be `icon`.